mobify/hora.js
# Adding grunt tasks for building dist

Added grunt tasks for building dist files.

**IMPORTANT - this branch needs to be merged before #4**

Status: **Ready to merge**

Reviewers: @ericmuyser 
## Changes
- Added npm modules for grunt tasks
- Added grunt tasks
- Built `dist`
## How to test-drive this PR
- checkout this branch
- `npm install`
- `grunt build`
- ensure file is built correctly
